Dojo.OpenApiGenerator/CodeTemplates/AbstractControllerTemplate.mustache

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.CodeDom.Compiler;
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Http;
+using {{ProjectNamespace}}.Services;
+using Dojo.OpenApiGenerator.Exceptions;
+
+namespace {{ProjectNamespace}}.Generated.Controllers{{#SourceCodeVersion}}.V{{SourceCodeVersion}}{{/SourceCodeVersion}}
+{
+    [ApiController]
+    {{#Version}}[ApiVersion("{{Version}}")]{{/Version}}
+    [GeneratedCode("Dojo.OpenApiGenerator", "{Assembly.GetExecutingAssembly().GetName().Version}")]
+    public abstract class {{Title}}ControllerBase : ControllerBase
```

Comment:
Didn't we agree on partial controllers last time? 🙂

Reply:
I had aditional discussions, and abstract will expose the controller action you need to implement.
Partial will not work because you will not have access to the method action you want to define response behavior for.
This will give full flexibility to the dev to control the logic inside each API action.